src/components/MovieSearch/SearchedList.tsx

```diff
@@ -1,25 +1,51 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import styled from 'styled-components';
+import { MovieProps } from 'Movies';
+import getFilteredMovies from '@/utils/recommend-movie-list';
 
 interface searchListProps {
-  inputText?: string;
+  inputText: string;
+  movies: MovieProps[];
 }
 
-const SearchedList = ({ inputText }: searchListProps) => {
-  const tempArray = [1, 2, 3];
+const SearchedList = ({ inputText, movies }: searchListProps) => {
+  const [recommendedMovies, setRecommendedMovies] = useState<string[]>([]);
+  const filteredOptions = {
+    inputText: inputText,
+    movies: movies,
+    listCount: 10,
+  };
+
+  useEffect(() => {
+    console.log('inputText', inputText);
+    setRecommendedMovies(getFilteredMovies(filteredOptions));
+  }, [inputText]);
+
+  const createMarkup = (htmlElement: string) => {
+    return { __html: htmlElement };
+  };
+
   return (
     <Wrapper>
       <GuideText>추천 검색어</GuideText>
-      <div>
+      <RecommendMoviesContainer>
         <ul>
-          {tempArray.map((num) => (
-            <RecommenedTextWrap key={num}>
-              <RecommendedText>{inputText}</RecommendedText>
+          {recommendedMovies.length === 0 ? (
+            <RecommenedTextWrap isEnabledHover={false}>
+              <RecommendedText>검색어 없음</RecommendedText>
```

Comment:
요런 식으로 사용자를 고려하도록 워딩을 수정하는 건 어떨까요? ux 라이팅!
```suggestion
              <RecommendedText>찾으려는 작품이 존재하지 않습니다.</RecommendedText>
```

Reply:
좋은 것 같아요. 반영해볼게요!